README.md

```diff
@@ -46,6 +46,8 @@ git merge dev
 ### Git Workflow
 > The main branch is used for production code only and should only be pushed to from dev branch after testing. The dev branch is used to stage changes from feature branches. Feature branches are used for all four of the main widgets while new branches off dev are opened for bug fixes.
 
+### Deploying with environment variables
+> By default, a browswer is unable to access the environment variables stored in a .env file. To make this information available, we need to install both 'dotenv' and 'process'. At the top of our webpack file, we will invoke the config command from dotenv. New environment plugins needed to be added to the webpack plugin array for each variable in the env file. To prevent 'process not defined' errors, we need to require the process module at the top of the webpack file. The webpack resolve needs to have an alias that matches 'process' to 'process/browser'. Lastly, we need to provide a new plugin to webpack, giving an object that matches 'process' to 'process/broswer'.
```

Comment:
instead of 'we need to install both 'dotenv' and 'process'' how about '`dotenv` and `process` libraries were installed'